clients/drcachesim/CMakeLists.txt

```diff
@@ -751,9 +751,10 @@ if (BUILD_TESTS)
              COMMAND tool.drcachesim.miss_analyzer_unit_test)
```

Comment:
> Fixes: #6040

We merged #6060 in so this would no longer fix it by itself: see https://github.com/DynamoRIO/dynamorio/issues/6040#issuecomment-1561293826